asistente_ladm_col/logic/quality_rules/qr_boundary_points_covered_plot_nodes.py

```diff
@@ -0,0 +1,117 @@
+"""
+/***************************************************************************
+                              Asistente LADM-COL
+                             --------------------
+        begin           : 2022-06-20
+        git sha         : :%H$
+        copyright       : (C) 2022 by Sergio Ramírez (SwissTierras Colombia)
```

Comment:
Add other authors

Reply:
fixed in https://github.com/SwissTierrasColombia/Asistente-LADM-COL/commit/da4771f482468a1bc493b6df80c868d9d3b23e45